miro_preprocessor/xml_to_json_run_task/main.tf

```diff
@@ -0,0 +1,21 @@
+# Lambda for publishing ECS service schedules to an SNS topic
+
+module "lambda_xml_to_json_run_task" {
+  source     = "../../terraform/lambda"
+  source_dir = "${path.module}/target"
+
+  name        = "xml_to_json_run_task"
+  description = "Run the Miro XML to JSON ECS task"
```

Comment:
nit: The more I read it, the more this description bugs me. Maybe "*Start* the Miro XML to JSON ECS task"? Just to make it clear this Lambda isn’t actually doing the work.